admin/base/settings/defaults.py

```diff
@@ -197,3 +198,23 @@
         ]
     },
 }
+
+# Keen.io settings in local.py
+KEEN_PROJECT_ID = 'CHANGEME'
+KEEN_READ_KEY = 'CHANGEME'
+KEEN_WRITE_KEY = 'CHANGEME'
```

Comment:
If these are going to be defined in OSF settings local.py, then we should point at osf settings.